### ISSUE TITLE
cloudscraper exact version requirement no longer working?

### ISSUE THREAD

**Issue by omicron-o** [grrttedwards/wow-addon-updater#170]
It seems to me that the exact version listen in setup.py is no longer working. I kept getting the error 
```
Failed to find addon version number for game version: GameVersion.retail, https://www.curseforge.com/wow/addons/details
```
for all my addons. After changing the version requirement for cloudscraper to >= 1.2.46 and updating cloudscraper to 1.2.56 it started working again.

**Comment by Mayo77** [comment]
Same issue.  how did you update cloudscraper?

**Comment by omicron-o** [comment]
I applied the following changes to `setup.py` and then updated all of those modules. I suspect just updating cloudscraper is enough though, but I stopped testing after I got it working as I was busy at the time.

```diff
diff --git a/setup.py b/setup.py
index 518c125..7744a65 100644
--- a/setup.py
+++ b/setup.py
@@ -55,16 +55,16 @@ setup(
         ]
     },
     install_requires=[
-        "beautifulsoup4==4.9.1",
+        "beautifulsoup4>=4.9.1",
         "certifi==2020.6.20",
         "chardet==3.0.4",
-        "cloudscraper==1.2.46",
+        "cloudscraper>=1.2.46",
         "idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
         "pyparsing==2.4.7; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'",
         "requests==2.24.0",
         "requests-toolbelt==0.9.1",
-        "soupsieve==2.0.1; python_version >= '3.5'",
-        "urllib3==1.25.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'",
+        "soupsieve>=2.0.1; python_version >= '3.5'",
+        "urllib3>=1.25.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4' and python_version < '4'",
     ],
     python_requires=">=3.8.0",
 )
```

**Comment by grrttedwards** [comment]
Hey, thank you for reporting. It looks like my automated tests also failed today, so I think this is going to hit everyone.

I'll experiment and try to push a new release tonight.

**Comment by omicron-o** [comment]
I'm not super familiar with the setup.py install_requirements, is it possible to ignore the bugfix part of the versions somehow? Perhaps something along the lines of `cloudscraper>=1.2.0 and cloudscraper < 1.3.0`.

That way these small fixes in the upstream modules can just be updated in the environment of the user without needing any changes on the wow-addon-updater side. Minor version changes can still be manually reviewed should they be needed.

**Comment by grrttedwards** [comment]
I try not to actually maintain the setup.py dependencies directly, and use a module that takes the pipenv lock file and transforms it into setup.py dependencies. After updating the lock file you can update it with `pipenv-update sync -d`. Anyways, I have put this branch up here: https://github.com/grrttedwards/wow-addon-updater/pull/172 but unfortunately wowace is having issues now, and the tests are failing...

**Comment by grrttedwards** [comment]
Published Release 1.8.1, thanks all
